### PR TITLE
Slim bridge

### LINKED ISSUE #177: Slim integration
Hi,

has anyone done any thinking on integrating php-di with the Slim web framework/thingie?

This is basically a Slim web app:

index.php:

```php
$app = new \Slim\Slim();

$app->get('/orders(/:page)', function ($page = 1) use ($app) {
    $app->render('orders.twig', array( 'page' => $page ));
});
```

I could do it hardcoded of course but was wondering if someone came up with something better.

### PR DESCRIPTION
**Pull request by mnapoli**
Fixes #177 

This PR contains the documentation and news article for the Slim integration.

### CODE REVIEW

File: doc/frameworks/slim.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+[Slim 3](http://www.slimframework.com/) is a micro-framework for web applications and APIs. This framework has been one of the first to integrate [container-interop](https://github.com/container-interop/container-interop) in its core. That means that *Slim can work with any container* out of the box.
+
+Replacing Slim's default container with PHP-DI is thus easy. However this bridge provides several additional features *on top of that*:
+
+- controllers as services, allowing dependency injection in controllers
+- intelligent parameter injections in controller
+
+### Controllers as services
+
+While you controllers can be simple closures, you can also **write them as classes and have PHP-DI instantiate them only when they are called**:
```
Comment (jdreesen) [inline doc/frameworks/slim.md]: Typo: While **your** controllers...

File: news/18-slim-bridge-released.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+While PSR-7 and middlewares are very interesting, what's more useful for us is the use of **container-interop**. That means Slim 3 can work with any dependency injection container.
+
+It is very easy to replace the default container (Pimple) with PHP-DI, but today we are releasing a "PHP-DI - Slim" bridge that goes a little further. **Read below for an introduction of what's possible with the PHP-DI bridge**.
+
+You can also read **[the full documentation for PHP-DI in Slim](../doc/frameworks/slim.md)**.
+
+### Controllers as services
+
+While you controllers can be simple closures, you can also **write them as classes and have PHP-DI instantiate them only when they are called**:
```
Comment (jdreesen) [inline news/18-slim-bridge-released.md]: Again: While **your** controllers...

**Comment by jdreesen** [comment]
Very nice! :)

**Comment by mnapoli** [comment]
Thanks for the review! It's fixed now.

**Pull request merged**